template_content/{% if ide_vscode %}.vscode{% endif %}/settings.json.jinja

```diff
@@ -24,7 +24,7 @@
       // Prevent default import sorting from running; Ruff will sort imports for us anyway
       "source.organizeImports": false
     },
-    "editor.defaultFormatter": null
+    "editor.defaultFormatter": "ms-python.black-formatter"
```

Comment:
This is an extension that is still in preview and at last review had some open issues that seemed a bit like show-stoppers. Are we sure this is the one we want?

If so, this line should be a jinja if/else based on whether user has opted to use black or not.

Also, we would need to put this in the recommended extensions json file.

Reply:
I am looking into the issues and see if it is solved. if it was, I will add them to the extensions and if statements as well.